pkg/cnab/provider/helpers.go

```diff
@@ -0,0 +1,112 @@
+package cnabprovider
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/deislabs/cnab-go/claim"
+	"github.com/deislabs/cnab-go/utils/crud"
+	"github.com/deislabs/porter/pkg/config"
+	"github.com/pkg/errors"
+)
+
+type TestDuffle struct {
+	*Duffle
+	*config.TestConfig
+	ClaimStore TestStore
+}
+
+type TestStore struct {
+	backingStore map[string][]byte
```

Comment:
I feel bad because you implemented all of this but I'm pretty sure we could have also used afero here and not written a test backing store at all. It would be an afero in memory file system from the test context (which we got from the constructor of the TestDuffle) and then the existing crud claim store.

I think that would be preferable to reimplementing another claim store? Let me know i you'd like to collaborate quick on how to use afero to do that.

Reply:
Oops!  Good call; using afero was the way to go.  This file has since been removed.